core/assets/sounds/cues/tongueLashing/data.json

```diff
@@ -4,25 +4,29 @@
   "series": "megamix",
   "cues": [
     {
-      "id": "tongueLashing/gulp",
+      "id": "tongueLashing/three",
```

Comment:
If you're changing IDs then you have to use `deprecatedIDs` to replace the IDs you changed so it can be loaded correctly for older versions.
Example for this cue:
```json
{
  "id": "tongueLashing/three",
  "deprecatedIDs": ["tongueLashing/gulp"]
  // other fields here
}
```